### ISSUE TITLE
Instructions for running Hugo locally

### ISSUE THREAD

**Issue by agardnerIT** [perses/website#31]
I'd like to start contributing to perses and part of the way I do this with projects is to learn and document as I go.

I often submit docs PRs to help others along the way.

I've cloned this repo but having trouble running the doc stack locally.

I get a blank page on `1313`.

I've tried various combinations of the `hugo` command in `docker-compose.yaml` but no dice.

I've also tried:

```
cd website/website
git submodule add https://github.com/thuliteio/doks.git themes/doks
```

If you can help me get the doc stack running, I'd be happy to contribute docs as I learn.

**Comment by Nexucis** [comment]
Yeah I think we need to spend more time on the website to make it easier to run it / build it.

For the moment if you want to help on the documentation on Perses, you can do it by submitting PR in the upstream repo. There is a folder [docs](https://github.com/perses/perses/tree/main/docs) where that contains all documentation about Perses.

**Comment by agardnerIT** [comment]
Thanks. Opened #32 .

Would you be open to reworking it in mkdocs? I'm happy to build a POC for a new doc stack and volunteer to maintain it?

**Comment by Nexucis** [comment]
not needed anymore thanks to you and the new website stack
